opentelemetry/src/context.rs

```diff
@@ -460,6 +460,15 @@ impl ContextStack {
             // The empty context is always at the bottom of the [`ContextStack`]
             // and cannot be popped, and the overflow position is invalid, so do
             // nothing.
+            otel_warn!(
+                name: "Context.PopInvalidPosition",
```

Comment:
```suggestion
                name: "Context.OutOfOrderDrop",
```

Trying to make this more user-friendly, see if this is better. (push/pop is inner implementation mostly...)

Reply:
I prefer this wording too; updated